torch/distributed/rpc/api.py

```diff
@@ -28,7 +28,7 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
-def join_rpc():
+def wait_all_workers():
```

Comment:
calling it `wait_all_workers` as per discussion, though for 1.4, it will probably still join all threads, which is more of a shutdown operation (meaning that user can't keep sending rpcs after this). maybe `join_all_workers` is more appropriate?